graf2d/primitivesv7/inc/ROOT/RTitle.hxx

```diff
@@ -0,0 +1,101 @@
+/*************************************************************************
+ * Copyright (C) 1995-2017, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT7_RTitle
+#define ROOT7_RTitle
+
+#include <ROOT/RDrawable.hxx>
+#include <ROOT/RAttrText.hxx>
+#include <ROOT/RPadPos.hxx>
+
+#include <memory>
+#include <string>
+
+namespace ROOT {
+namespace Experimental {
+
+/** \class ROOT::Experimental::RTitle
+\ingroup GrafROOT7
+\brief A title for the RFrame.
+\author Sergey Linev <s.linev@gsi.de>
+\date 2020-02-26
+\warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback is welcome!
+*/
+
+
+class RTitle : public RDrawable {
```

Comment:
`RFrameTitle`? "Title" could be anything. Or is it meant to also be the axis title, the canvas title, etc? I guess not...

Reply:
For the moment it is not that important - I can rename it to `RFrameTitle`.
I just want to introduce class which does not use absolute position, but relative to something else. And test complete tool chain - object, attributes, display in JS, interactivity, ....
More interesting are attributes for `RTitle` - they can be reused for different components. Like axis title.